pop_test.go

```diff
@@ -148,13 +148,13 @@ type Enemy struct {
 }
 
 type Song struct {
-	ID         uuid.UUID `db:"id"`
-	Title      string    `db:"title"`
-	UserID     int       `db:"u_id"`
-	CreatedAt  time.Time `json:"created_at" db:"created_at"`
-	UpdatedAt  time.Time `json:"updated_at" db:"updated_at"`
-	ComposerID int       `json:"composer_id" db:"composer_id"`
-	ComposedBy Composer  `belongs_to:"composer"`
+	ID           uuid.UUID `db:"id"`
+	Title        string    `db:"title"`
+	UserID       int       `db:"u_id"`
+	CreatedAt    time.Time `json:"created_at" db:"created_at"`
+	UpdatedAt    time.Time `json:"updated_at" db:"updated_at"`
+	ComposedByID int       `json:"composer_id" db:"composer_id"`
```

Comment:
@larrymjordan there's some dissonance here now between the field name and the db/json names. To fix it would require a migration to change the field name in the test data. It didn't seem totally worth the hassle, but I'm happy to do it if you think it is.

Reply:
@stangah nice catch. It is not necessary to create a migration though. You just simply change 
```go

create_table("songs", func(t) {
  t.Column("composer_id", "int", {"null":true})
})
```
by

``` go
  t.Column("compose_by_id", "int", {"null":true})
```

in the migration file inside `migrations` folder.

Reply:
oh yea, I guess there's no reasons to not edit the existing migrations :)